src/lib.rs

```diff
@@ -434,61 +501,61 @@ make_color! {
     ///Linear luminance.
     Luma {
         ///Linear luminance.
-        y(luma: T)[alpha: T] => new;
+        y(luma: T)[alpha: T] => with_wp;
```

Comment:
I'm talking about ignoring `with_wp` for now and always assume D65 in the constructors. These things will change even more, as you said, so let's make it reasonably simple for ourselves.